js/modules/k6/http/http_request.go

```diff
@@ -109,9 +117,14 @@ func (h *HTTP) request(ctx context.Context, rt *goja.Runtime, state *common.Stat
 		URL:    url.URL,
 		Header: make(http.Header),
 	}
+	HTTPReq := &HTTPRequest{
+		Method: req.Method,
+		URL:    req.URL,
```

Comment:
Hm, I think this should be `req.URL.URLString` to be consistent with `res.url` (which is just a string). We should expose URL parsing/joining/composing functionality in a specific module though.